pipenv/utils.py

```diff
@@ -260,7 +261,8 @@ def convert_deps_to_pip(deps, r=True):
 
         # Support for files.
         if 'file' in deps[dep]:
-            dep = deps[dep]['file']
+            dep = '-e {0}'.format(deps[dep]['file'])
+            print('Adding dep: {}'.format(dep))
```

Comment:
we don't use print, anywhere.

Reply:
oh yes, i was using that for debugging...